Sheepish-Squirrels/bangazon-api-sprint1
# Krpayment

Changes proposed in this pull request:

-set up function to get all payment types, route to see all payment types, to start, test, reset data, and generate data
-wrote function to get one payment type using route params
- Also, added post one payment type function
used PUT to edit payment types
delete functionality on payment types
Steps to test:

In terminal,
run nodemon app.js
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
Make sure you can GET all payment types, GET one payment types, POST a payment type, PUT (edit) a payment type, and DELETE a payment type
